Resources/config/routing/posts.yaml

```diff
@@ -0,0 +1,7 @@
+orbitale_cms_posts:
+  path: /{date}/{slugs}
+  requirements:
+    slugs: "([a-zA-Z0-9_-]+\/?)*"
```

Comment:
Please use 4-spaces indentation in yaml files

Reply:
Got it

Reply:
This one is because my tab dosen't work